_opensearch/install/ansible.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Ansible playbook
+parent: Install OpenSearch
+nav_order: 60
+---
+
+# Ansible playbook
+
+You can use an Ansible playbook to install and configure a production-ready OpenSearch cluster along with OpenSearch Dashboards.
+
+The Ansible playbook only supports deployment of OpenSearch and OpenSearch Dashboards to CentOS7 hosts.
+{: .note }
```

Comment:
Is this typo?

Reply:
{: .note } makes the above sentence show up as a note.